src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,61 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("1,2을 ,로 split 했을 때 1과 2로 분리된다.")
+    public void splitStringWithComma() {
+        String input = "1,2";
+        String[] actual = input.split(",");
+        String[] expected = new String[]{"1", "2"};
+
+        assertThat(actual).containsExactly(expected);
```

Comment:
```suggestion
        String[] actual = "1,2".split(",");
        assertThat(actual).containsExactly("1", "2");
```
로컬 변수를 최소화하는 방향으로 구현해 보는 것은 어떨까?